twitcher/postgresdb.py

```diff
@@ -1,6 +1,54 @@
-from magpie.db import get_session_factory, get_engine, get_tm_session
```

Comment:
Wasn't this better?
If postgres is only used with Magpie, it duplicates the code.

Reply:
Agree. I think that was an effort to remove magpie dependency :(
But agin postpone to magpie auth refactoring